src/main.js

```diff
@@ -212,6 +216,31 @@ function showNewView() {
   saveView.classList.add("hidden")
 };
 
+//Save poster function
+
+function savePoster() {
```

Comment:
This function seems to have less to do with saving posters, and more to do with displaying them.

Reply:
Great point. I have updated this function name to be displayPoster since that's more clear.